src/app/containers/AudioVideo/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import deepGet from '../../lib/utilities/deepGet';
+import Canonical from './Canonical';
+import Caption from '../Caption';
+import videoMetadata from './audioVideoMetadata';
+import {
+  NestedGridParentLarge,
+  NestedGridParentSmall,
+  NestedGridItemChildSmall,
+  NestedGridItemChildLarge,
+  GridItemConstrainedLargeNoMargin,
+  GridItemConstrainedSmall,
+} from '../../lib/styledGrid';
+
+import {
+  audioVideoPropTypes,
+  emptyBlockArrayDefaultProps,
+} from '../../models/propTypes';
+import filterForBlockType from '../../lib/utilities/blockHandlers';
+import { RequestContext } from '../../contexts/RequestContext';
+
+const selectWrappers = orientation => {
+  const wrapperSpan = {
+    default: '6',
+    group5: '12',
+  };
+  let ParentWrapper = NestedGridParentLarge;
+  let ChildWrapper = NestedGridItemChildLarge;
+  let Container = GridItemConstrainedLargeNoMargin;
+  let portrait = false;
+
+  if (orientation === 'Portrait') {
```

Comment:
Does `orientation` always come through with `Portrait` capitalised? Should we force it all to lowercase to be safe?

Reply:
I think it is a safe assumption that it is capitalised.